db/db_with_timestamp_basic_test.cc

```diff
@@ -435,6 +435,32 @@ TEST_F(DBBasicTestWithTimestamp, UpdateFullHistoryTsLow) {
   Close();
 }
 
+TEST_F(DBBasicTestWithTimestamp, UpdateFullHistoryTsLowWithPublicAPI) {
```

Comment:
Add some tests in this test fixture for corner cases in which:
- increase ts_low for a column family that does not enable timestamp
- call `GetFullHistoryTsLow()` for a column family disabling timestamp
- call `IncreaseFullHistoryTsLow()` with a timestamp whose length is longer than the cf's timestamp size.
- call `IncreaseFullHistoryTsLow()` with empty `ts_low`

Reply:
added